pkg/handlers/webhookreceiver.go

```diff
@@ -148,15 +146,24 @@ func (h *WebhookReceiverHandler) processAlert(alert template.Alert, mnl *oav1alp
 	}
 	// Send the servicelog for the alert
 	log.WithFields(log.Fields{LogFieldNotificationName: notification.Name}).Info("will send servicelog for notification")
-	err = ocm.BuildAndSendServiceLog(
-		ocm.NewServiceLogBuilder(notification.Summary, notification.ActiveDesc, notification.ResolvedDesc, viper.GetString(config.ExternalClusterID), notification.Severity, notification.LogType, notification.References),
-		firing, &alert, h.ocm)
-	if err != nil {
-		log.WithError(err).WithFields(log.Fields{LogFieldNotificationName: notification.Name, LogFieldIsFiring: true}).Error("unable to send a notification")
+
+	var attempts int = 5
```

Comment:
WDYT about including the number of attempts and sleep duration in the ocm-agent ConfigMap?

Reply:
@chamalabey Thanks for the review. This is something I do think that these parameters shouldn't be exposed externally. I couldn't of any scenario where we want this to be configurable for different use cases?

Reply:
My thinking was not hard code the values in the code and have them in a configmap so they can be configured without code changes. If you think this is not going to be changed for any reason, it is okay to hard code it.